scrapers/SweetyX.yml

```diff
@@ -0,0 +1,28 @@
+name: SweetyX
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sweetyx.com/en/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $wrapper: //div[@class="article_contents"]/div[@class="container"]/div[@class="video_description"]/div[@class="row"]/div[@class="col-sm-9"]
+    scene:
+      Title: //title/text()
+      Details: $wrapper/p/text()
```

Comment:
the Details part has two issues.
1. You are getting only the fist paragraph of the description instead of all
2. In most of the scenes the Description has text from anchors or crossed out or text not directly in the paragraph

```yaml
      Details:
        selector: $wrapper/p
        concat: "\n\n"
```
reloves both issues